docs/content/preview/explore/ysql-language-features/advanced-features/inheritance.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Table inheritance
+linkTitle: Table inheritance
+description: Table inheritance in YSQL
+menu:
+  preview:
+    identifier: advanced-features-inheritance
+    parent: advanced-features
+    weight: 400
+type: docs
+---
+
+
+## Overview
+
+[BETA] YSQL supports table inheritance, which is a [PostgreSQL feature](https://www.postgresql.org/docs/current/ddl-inherit.html) that allows users to create child tables that inherit columns and certain constraints from one or more parent tables.  
```

Comment:
This beta doesn't show up correctly on the preview - how do we mark this feature as a beta SQL feature? We can call it tech preview if beta is no longer the right word